configs/components/openssl.rb

```diff
@@ -54,6 +66,23 @@
     ldflags = ''
     cflags = "$${CFLAGS} -static-libgcc"
     pkg.environment "CC" => "/opt/pl-build-tools/bin/gcc"
+  elsif platform.is_windows?
+    if platform.architecture == "x64"
```

Comment:
arch is no longer needed, this could be replaced with `target = platform.architecture == "x64" ? "mingw64" : "mingw"`